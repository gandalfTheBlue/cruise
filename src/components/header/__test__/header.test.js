import React from 'react'
import renderer from 'react-test-renderer'

import Header from '../header'

it('renders correctly header content', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})
