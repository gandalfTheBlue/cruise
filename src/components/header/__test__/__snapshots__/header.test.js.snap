// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly header content 1`] = `
<div
  className="header"
>
  <div
    className="header-container"
  >
    <div />
    <img
      alt="logo"
      src="/assets/logo/logo.svg"
    />
    <div>
      <div
        className="header-avatar"
        onClick={[Function]}
      >
        <img
          alt="avatar"
          src="/assets/avatar/yaoming_meme.svg"
        />
        <span
          className="iconfont icon-angle-down"
        />
      </div>
      <ul
        className="user-option"
      />
    </div>
  </div>
</div>
`;
